### ISSUE TITLE
[Feature request] Task description

### ISSUE THREAD

**Issue by engibengi** [sioodmy/todo#21]
A cool addition would be the possiblity to add a description to a task, to further specify the todo goal.

**Comment by dkmar** [comment]
agree. Having something like a `todo note [index] [note]` command to add notes or thoughts associated with a todo is probably the only thing i really want added. The alternative is starting some document in obsidian or whatever for describing the task, and if you're doing that then it might make sense to do all tasks there as well. I'd rather use `todo`
